### PR TITLE
pushing a number of resque logs deeper

### PR DESCRIPTION
**Pull request by gcoonrod**
Trying to reduce the verbosity of resque logging.

### CODE REVIEW

**Comment by evantahler** [comment]
I'm personally of the opinion that starging/stopping workers should be of the `info` level, but lets here what others think

**comments welcome**

**Comment by gcoonrod** [comment]
When using multiworkers i was seeing workers stopped and started every second. Which really cluttered up the logs. What is the rationale for having those events at the info level?

**Comment by evantahler** [comment]
The rationale was in production the lowest log level you probably might use would be "info".  On production you are likely to be using a log aggression tool like Splunk, Sumologic, etc, and you probably want to know how many of your workers are running at any time.  You can then scan for the latest log of this type to see how many are up and running.

**Comment by neilstuartcraig** [comment]
Info seems like the right choice to me too. It's certainly not higher than that. I guess you could make a case for debug but I'd expect to see it in info. 
For reference, you could check what apps like nginx etc do. Not saying they're becessarily right but could be validation.

**Comment by gcoonrod** [comment]
Any thoughts on making the worker log levels controllable?

**Comment by evantahler** [comment]
lets see what a PR for that would look like @gcoonrod! Might make sense depending on how much cruft that adds to the config file

**Comment by gcoonrod** [comment]
@evantahler, I added an attribute to the tasks config and set some of the node-resque messages to use that attribute to determine their logging level. I only applied that option to the messages that have been bugging me. 

Are there other messages that this option could/should be applied to?

File: config/tasks.js [hunk 1]
```diff
@@ -2,16 +2,18 @@ exports.default = {
   tasks: function(api){
     return {
       // Should this node run a scheduler to promote delayed tasks?
-      scheduler: false,
+      scheduler: true,
```
Comment (evantahler) [inline config/tasks.js]: Don't commit this

File: config/tasks.js [hunk 2]
```diff
@@ -2,16 +2,18 @@ exports.default = {
...
       // what queues should the taskProcessors work?
       queues: ['*'],
+	  // Verbosity of task logging
+	  verbose: true,
       // how long to sleep between jobs / scheduler checks
       timeout: 5000,
       // at minimum, how many parallel taskProcessors should this node spawn?
       // (have number > 0 to enable, and < 1 to disable)
-      minTaskProcessors: 0,
+      minTaskProcessors: 1,
```
Comment (evantahler) [inline config/tasks.js]: and don't commit these.  we should start default projects with no workers

**Comment by gcoonrod** [comment]
@evantahler Any further comments or suggestions?

File: config/tasks.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ exports.default = {
       scheduler: false,
       // what queues should the taskProcessors work?
       queues: ['*'],
+      // Verbosity of task logging
+      verbose: true,
```
Comment (evantahler) [inline config/tasks.js]: I don't think that I like having a log option outside of the logging config file.

Perhaps the answer is a hash with each event => log level?

```js
{start: "debug", poll: "trace"}
```

**Comment by evantahler** [comment]
Lets merge it in for now, although I do expect some future changes.

**Comment by evantahler** [comment]
Thanks!

**Pull request merged**